src/main/java/Flaaaash.java

```diff
@@ -0,0 +1,256 @@
+                            deadline.markAsDone();
+                        }
+                        break;
+                    case "E":
+                        Event event = new Event(description, parts[3], parts[4]);
+                        addTaskNoText(event);
+                        if (isDone) {
+                            event.markAsDone();
+                        }
+                        break;
+                    default:
+                        throw new IllegalArgumentException("Invalid task type");
+                    }
+
+                } catch (Exception e) {
+                    System.out.println("Skipping corrupted entry: " + line);
+                }
+            }
+            
+            fileScanner.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found, starting fresh.");
+        }
+    }
+
+    private static void markTask(int index) {
+        System.out.println("____________________________________________________________");
+        if (index < 0 || index > tasks.size()) {
+            System.out.println(" Invalid index");
+        } else {
+            System.out.println(" Nice! I've marked this task as done:");
+            tasks.get(index - 1).markAsDone();
+            System.out.println("    " + tasks.get(index - 1));
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void unmarkTask(int index) {
+        System.out.println("____________________________________________________________");
+        if (index < 0 || index > tasks.size()) {
+            System.out.println(" Invalid index");
+        } else {
+            System.out.println(" OK, I've marked this task as not done yet:");
+            tasks.get(index - 1).unmarkAsDone();
+            System.out.println("    " + tasks.get(index - 1));
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void addTask(Task task) {
+        tasks.add(task);
+
+        System.out.println("____________________________________________________________");
+        System.out.println(" Got it. I've added this task:");
+        System.out.println("    " + task);
+        System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void deleteTask(int index) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Noted. I've removed this task:");
+        System.out.println("  " + tasks.get(index - 1));
+        tasks.remove(tasks.get(index - 1));
+        System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+        System.out.println("____________________________________________________________");
```

Comment:
Avoid repeating this code line. Store it in a variable or make it a function and use it multiple times.